simplejira/prompt.py

```diff
@@ -460,3 +461,19 @@ def do_editwork(self, args):
         collection = worklog_collection(worklogs)
         print(editor_ignore_comments(collection.to_yaml()))
     '''
+    assignee_parser = argparse.ArgumentParser()
+    assignee_parser.add_argument('assignee', const=None, type=str,
+                                 help="Assign card to someone as assign <username>;"
+                                 "username is case insensitive")
+    @cmd2.with_argparser(assignee_parser)
+    def do_assign(self, args):
+        if not args.assignee:
+            args.assignee = self.input("Enter assignee user id:")
+        continue_assignment = 'Y'
+        if not args.assignee:
+            continue_assignment = self.input("Leaving assignee blank would unassign the card,"
```

Comment:
ah so for this, you can use 'prompter.yesno("Question")' and it'll automatically put the 'Y/n' and return true/false for you. So you can just do

```python
continue_assignment = prompter.yesno("Leaving assignee blank would unassign the card. Continue?")
if continue_assignment:
    stuff
else:
    stuff